gcp_variant_transforms/beam_io/vcf_estimate_io.py

```diff
@@ -20,35 +20,26 @@
 from functools import partial
 from typing import Dict, Iterable  # pylint: disable=unused-import
 
+from apache_beam import transforms
 from apache_beam.io import filebasedsource
 from apache_beam.io import range_trackers  # pylint: disable=unused-import
-from apache_beam.io.filesystem import CompressionTypes
-from apache_beam.io.filesystem import CompressedFile
-from apache_beam.io.filesystems import FileSystems
-from apache_beam.io.iobase import Read
-from apache_beam.transforms import PTransform
+from apache_beam.io import filesystem
+from apache_beam.io import filesystems
+from apache_beam.io import iobase
 
 
 class VcfEstimate(object):
   """Container for estimation data about the VCF file."""
 
   def __init__(self,
-               file_name=None,  # type: str
+               file_name,  # type: str
                estimated_line_count=None,  # type: float
```

Comment:
Since they have default value (None), does it mean we may skip evaluating some of these?